<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FVAClientUI</class>
 <widget class="QMainWindow" name="FVAClientUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Cliente FVA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icono.png</normaloff>:/images/icono.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>-1.000000000000000</double>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QTabWidget" name="usrSlots">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(76, 118, 82);
background-color:rgb(216, 230, 225);</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <attribute name="title">
        <string>Dispositivo</string>
       </attribute>
       <layout class="QGridLayout" name="tab1Layout">
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="refreshDevices">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(11, 35, 21);
font: 75 9pt &quot;Noto Sans&quot;;
background-color: rgb(229, 229, 229);
</string>
          </property>
          <property name="text">
           <string>Refrescar Dispositivos</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/images/refrescar.png</normaloff>:/images/refrescar.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>100</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="initialMessageLbl">
          <property name="styleSheet">
           <string notr="true">font: 13pt;</string>
          </property>
          <property name="text">
           <string>Favor conecte su dispositivo de firma para comenzar.</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuRequests">
    <property name="title">
     <string>Solicitar</string>
    </property>
    <addaction name="actionSign"/>
    <addaction name="actionAuthentication"/>
   </widget>
   <widget class="QMenu" name="menuContactos">
    <property name="title">
     <string>Contactos</string>
    </property>
    <addaction name="actionManageContacts"/>
    <addaction name="actionManageGroups"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionMySignatures"/>
    <addaction name="actionMyRequests"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFrima">
    <property name="title">
     <string>Mis Documentos</string>
    </property>
    <addaction name="actionFirmar_Autenticar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRequests"/>
   <addaction name="menuFrima"/>
   <addaction name="menuContactos"/>
  </widget>
  <action name="actionSign">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/firma.png</normaloff>:/images/firma.png</iconset>
   </property>
   <property name="text">
    <string>Firma</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAuthentication">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/autenticacion.png</normaloff>:/images/autenticacion.png</iconset>
   </property>
   <property name="text">
    <string>Autenticaci√≥n</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionManageContacts">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/administrar_contactos.png</normaloff>:/images/administrar_contactos.png</iconset>
   </property>
   <property name="text">
    <string>Administar Contactos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionManageGroups">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/administrar_grupos.png</normaloff>:/images/administrar_grupos.png</iconset>
   </property>
   <property name="text">
    <string>Administrar Grupos</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/configuracion_modulo.png</normaloff>:/images/configuracion_modulo.png</iconset>
   </property>
   <property name="text">
    <string>Preferencias</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionMySignatures">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/mis_firmas.png</normaloff>:/images/mis_firmas.png</iconset>
   </property>
   <property name="text">
    <string>Mis Firmas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionMyRequests">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/mis_solicitudes.png</normaloff>:/images/mis_solicitudes.png</iconset>
   </property>
   <property name="text">
    <string>Mis Solicitudes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/salir.png</normaloff>:/images/salir.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFirmar_Autenticar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/firma.png</normaloff>:/images/firma.png</iconset>
   </property>
   <property name="text">
    <string>Firmar - Autenticar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
